src/include/cache.i

```diff
@@ -68,6 +79,21 @@ __wt_cache_read_gen_new(WT_SESSION_IMPL *session, WT_PAGE *page)
 }
 
 /*
+ * __wt_cache_stuck --
+ *      Indicate if the cache is stuck (i.e., not making progress).
+ */
+static inline bool
+__wt_cache_stuck(WT_SESSION_IMPL *session)
+{
+	WT_CACHE *cache;
+
+	cache = S2C(session)->cache;
+	return (cache->evict_aggressive_score == WT_EVICT_SCORE_MAX &&
```

Comment:
How about 200ms / 2s?

There is a balancing act -- we don't want to start forcing transactions to roll back unnecessarily but if the lookaside table is really required to make progress with some workload, it would be nice to figure that out without waiting too long.